data.sql

```diff
@@ -64,4 +64,161 @@ UPDATE animals
 SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
 WHERE name = 'Angemon' OR name = 'Boarmon';
 
-COMMIT;
\ No newline at end of file
+COMMIT;
+
+-- Join visits
+
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('William Tatcher',45,'2000-04-23');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Maisy Smith',26,'2019-01-17');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Stephanie Mendez',64,'1981-5-04');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Jack Harkness',38,'2008-6-08');
+
+INSERT INTO specializations (vets_id,species_id) VALUES (1,1);
+INSERT INTO specializations (vets_id,species_id) VALUES (3,1);
+INSERT INTO specializations (vets_id,species_id) VALUES (3,2);
+INSERT INTO specializations (vets_id,species_id) VALUES (4,2);
```

Comment:
* Kindly fix this error I get when I run these queries, below 👇 is a screenshot of the error I get 
![image](https://user-images.githubusercontent.com/52497246/213638762-be9b394a-52e9-4fd5-a098-ddb2586fc6ba.png)